custom/panel_templates/Default/template.php

```diff
@@ -16,11 +16,17 @@
 if (!class_exists('Default_Panel_Template')) {
     class Default_Panel_Template extends TemplateBase {
 
-        private Language $_language;
+        // Private variable to store user
+        private $_user, $_pages;
+
+        /** @var Language */
+        private $_language;
 
         // Constructor - set template name, version, Nameless version and author here
-        public function __construct(Smarty $smarty, Language $language) {
+        public function __construct($cache, $smarty, $language, $user, $pages) {
```

Comment:
Not sure what happened here!
We are using typehints now, and you seem to have added an unused`$cache` variable...